### PR TITLE
[DOC-2751] Manage SMP feature flags

### PR DESCRIPTION
**Pull request by brian-f-harness**
Manage SMP feature flags

For reviewers, [a preview](https://649378777d4eb121f9a07e96--harness-developer.netlify.app/docs/self-managed-enterprise-edition/self-managed-helm-based-install/manage-feature-flags) is available.

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by pratmit — APPROVED** [review]
LGTM

File: docs/self-managed-enterprise-edition/self-managed-helm-based-install/manage-feature-flags.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+To view installed Feature Flags, do the following:
+
+1. Open your global `override-prod.yaml` file in a text editor.
+
+   :::info note
+   This file is typically located in the following default path.
+   :::
+
+   ```
+   /path/to/helm-charts/src/override-prod.yaml
```
Comment (SudheendraKatte) [inline docs/self-managed-enterprise-edition/self-managed-helm-based-install/manage-feature-flags.md:22]: The path should have been inside the note. Please consider.

File: docs/self-managed-enterprise-edition/self-managed-helm-based-install/manage-feature-flags.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+To add Feature Flags, do the following:
+
+1. Open your global `override-prod.yaml` file in a text editor.
+
+   :::info note
+   This file is typically located in the following default path.
+   :::
+
+   ```
+   /path/to/helm-charts/src/override-prod.yaml
```
Comment (SudheendraKatte) [inline docs/self-managed-enterprise-edition/self-managed-helm-based-install/manage-feature-flags.md:40]: Path is outside the Note section.

File: docs/self-managed-enterprise-edition/self-managed-helm-based-install/manage-feature-flags.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+   ```
+   helm upgrade <my-release> . -n harness-ng -f ../override-prod.yaml
+   ```
+   
+   :::info note
+   For Feature Flags that affect `ng-manager`, you must restart the component(s) after the Helm upgrade is complete and the Harness Manager has restarted and is up and running. You can restart your pods or run the following rollout restart command on the Harness Manager deployment for your installation.
+   :::
+
+   ```
+   kubectl rollout restart deployment ng-manager
```
Comment (SudheendraKatte) [inline docs/self-managed-enterprise-edition/self-managed-helm-based-install/manage-feature-flags.md:116]: Please check if this should have been inside the Note section.

**Review by SudheendraKatte — DISMISSED** [review]
Approved with minor comments.

File: docs/self-managed-enterprise-edition/self-managed-helm-based-install/manage-feature-flags.md (hunk 2 above)
Comment (brian-f-harness) [reply docs/self-managed-enterprise-edition/self-managed-helm-based-install/manage-feature-flags.md:41]: Updated. Nice catch, thank you.

File: docs/self-managed-enterprise-edition/self-managed-helm-based-install/manage-feature-flags.md (hunk 3 above)
Comment (brian-f-harness) [reply docs/self-managed-enterprise-edition/self-managed-helm-based-install/manage-feature-flags.md:117]: Updated. Nice catch, thank you.

File: docs/self-managed-enterprise-edition/self-managed-helm-based-install/manage-feature-flags.md (hunk 1 above)
Comment (brian-f-harness) [reply docs/self-managed-enterprise-edition/self-managed-helm-based-install/manage-feature-flags.md:21]: Updated. Nice catch, thank you.

**Pull request merged**